docs/install.md

```diff
@@ -135,23 +140,23 @@ You can use `--default-cidr` flags below to config default Pod CIDR or create a
       --alsologtostderr                   log to standard error as well as files
       --bind-socket string                The socket daemon bind to. (default "/run/openvswitch/kube-ovn-daemon.sock")
       --default-interface-name string     The default host interface name in the vlan/vxlan type
-      --default-provider-name string      The vlan or vxlan type default provider interface name, default: provider (default "provider")
-      --enable-mirror                     Enable traffic mirror, default: false
-      --encap-checksum                    Enable checksum, default: true (default true)
-      --iface string                      The iface used to inter-host pod communication, can be a nic name or a group of regex separated by comma, default: the default route iface
+      --default-provider-name string      The vlan or vxlan type default provider interface name (default "provider")
+      --enable-mirror                     Enable traffic mirror
```

Comment:
enable-mirror   the default value should be added

Reply:
Added in flag description.